app/models/round.rb

```diff
@@ -58,8 +58,18 @@ def format
     self.number = 0
   end
 
+  # Competitions before 2026 have to use Mo3 for 333bf, but after 2026 they need to use Ao5
+  NEW_REGULATIONS_START_DATE = Date.new(2026, 1, 1)
```

Comment:
"New Regulations" is way too generic as a name for this field

Reply:
ideal would `2026_REGULATIONS...` but we can't start with a numeral so I did `REGULATIONS_2026...`